apps/web/data/classes/commander.ts

```diff
@@ -221,15 +199,41 @@ const commanderFeatures: ClassFeature[] = [
     description: "Gain the Coordinated Strike! Commander's Order.",
     traits: [
       {
-        id: "commander-coordinated-strike-0",
+        id: "coordinated-strike-pool",
+        type: "resource",
+        resourceDefinition: {
+          id: "coordinated-strike-pool-0",
```

Comment:
the `-0` suffix is usually used for the traits, not the resource. Maybe call the resource `coordinated-strikes` instead